src/components/ApplicationGroup/Details/EnvironmentOverview/EnvironmentOverview.tsx

```diff
@@ -82,61 +166,292 @@ export default function EnvironmentOverview({
-                            {!isVirtualEnv && <span>{OVERVIEW_HEADER.APP_STATUS}</span>}
-                            <span>{OVERVIEW_HEADER.DEPLOYMENT_STATUS}</span>
-                            <span>{OVERVIEW_HEADER.LAST_DEPLOYED}</span>
+                    <div className="app-deployments-info-wrapper dc__overflow-scroll w-100 dc__position-rel min-w-500">
+                        <div
+                            className={`app-deployments-info-header display-grid dc__align-items-center dc__border-bottom-n1 dc__uppercase fs-12 fw-6 cn-7 ${lastDeployedClassName}`}
+                        >
+                            <div className="pl-16 pr-16 app-deployment-info-row-leftsection display-grid dc__position-sticky sticky-column bcn-0 h-100 dc__align-items-center">
+                                <label className="dc__position-rel pointer m-0-imp">
+                                    <input
+                                        type="checkbox"
+                                        className="form__checkbox"
+                                        value={'ALL'}
+                                        onChange={handleSelect}
+                                        checked={selectedAppIds.length === appListData.appInfoList.length}
+                                    />
+                                    <span
+                                        className={`form__checkbox-container ${
+                                            selectedAppIds.length === appListData.appInfoList.length
+                                                ? 'tick-icon'
+                                                : selectedAppIds.length > 0
+                                                ? 'any-selected'
+                                                : ''
+                                        }`}
+                                    ></span>
+                                </label>
+                                <span className="">{!isVirtualEnv && <ActivityIcon className="icon-dim-16" />}</span>
+                                <span className="">{OVERVIEW_HEADER.APPLICATION}</span>
+                            </div>
+                            <span className="">{OVERVIEW_HEADER.DEPLOYMENT_STATUS}</span>
+                            <button
+                                type="button"
+                                className="dc__outline-none-imp p-0 dc__uppercase dc__transparent flexbox dc__align-items-center dc__gap-4"
+                                onClick={toggleIsLastDeployedExpanded}
+                            >
+                                <span className="">{OVERVIEW_HEADER.LAST_DEPLOYED}</span>
+                                <ArrowLineDown
+                                    className="icon-dim-14 scn-5 rotate"
+                                    style={{ ['--rotateBy' as any]: isLastDeployedExpanded ? '90deg' : '-90deg' }}
+                                />
+                            </button>
```

Comment:
any specific reason for the empty classname

Reply:
no, removing